converters/toZigbee.js

```diff
@@ -351,8 +351,12 @@ const converters = {
                             {brightness: meta.state.brightness, turnedOffWithTransition: true};
                     }
 
-                    const level = state === 'off' ? 0 :
+                    let level = state === 'off' ? 0 :
                         (store[entity.deviceIeeeAddress] ? store[entity.deviceIeeeAddress].brightness : 255);
```

Comment:
If brightness comes from the store we can assume it's max 254 (as we control what to set in it). `store[entity.deviceIeeeAddress].brightness : 255);` should be changed to  `store[entity.deviceIeeeAddress].brightness : 254);` and the rest of the changes can be undone.